test/docfx.Test/lib/JsonSchemaTest.cs

```diff
@@ -253,6 +253,14 @@ public void TestJsonSchemaConfirmance(string description, string schemaText, str
             "['warning','field-deprecated','Deprecated field: 'key1'.','file',1,10]")]
         [InlineData("{'properties': {'key1': {'replacedBy': 'key2'}}}", "{'key1': 1}",
             "['warning','field-deprecated','Deprecated field: 'key1', use 'key2' instead','file',1,10]")]
+
+        // enum dependencies validation
+        [InlineData("{'properties': {'key1': {'type': 'string', 'enum': ['.net', 'yammer']}, 'key2': {'type': 'string'}}, 'enumDependencies': { 'key2': { 'key1': { '.net': ['', 'csharp', 'devlang'], 'yammer': ['', 'tabs', 'vba']}}}}", "{'key1': 'yammer', 'key2': 'tabs'}", "")]
+        [InlineData("{'properties': {'key1': {'type': 'string', 'enum': ['.net', 'yammer']}, 'key2': {'type': 'string'}}, 'enumDependencies': { 'key2': { 'key1': { '.net': ['', 'csharp', 'devlang'], 'yammer': ['', 'tabs', 'vba']}}}}", "{'key1': 'yammer'}", "")]
```

Comment:
>[InlineData("{'properties': {'key1': {'type': 'string', 'enum': ['.net', 'yammer']}, 'key2': {'type': 'string'}}, 'enumDependencies': { 'key2': { 'key1': { '.net': ['', 'csharp', 'devlang'], 'yammer': ['', 'tabs', 'vba']}}}}", "{'key1': 'yammer'}", "")] [](start = 8, length = 253)

what should the behavior for the following case? 
```
[InlineData("{'properties': {'key1': {'type': 'string', 'enum': ['.net', 'yammer', 'undefined']}, 'key2': {'type': 'string'}}, 'enumDependencies': { 'key2': { 'key1': { '.net': ['', 'csharp', 'devlang'], 'yammer': ['', 'tabs', 'vba']}}}}", "{'key1': 'undefined', 'key2': 'tabs'}", "")]
```
does user have to specify `enumDependencies` for all enum values?

Reply:
Good question.For current logic, this will cause lack dependency error

Reply:
Good question.For now logic, no error will be return. Because when the user does not specify a correspondence, ‘’key2 can theoretically be an arbitrary value.